easybuild/easyconfigs/n/nose-parameterized/nose-parameterized-0.5.0-CrayGNU-17.08-python3.eb

```diff
@@ -15,15 +19,15 @@ source_urls = [PYPI_SOURCE]
 sources = [SOURCE_TAR_GZ]
 
 dependencies = [
-             ('cray-python/17.06.1', EXTERNAL_MODULE),
+             ('cray-python/%s.1.1' % pyver, EXTERNAL_MODULE),
```

Comment:
I suggest to define the complete module version together with req_py_majver, req_py_majver and pyver and just use the new variable here (something like `py_mod_ver`)